packages/twenty-front/src/modules/page-layout/widgets/components/WidgetContentRenderer.tsx

```diff
@@ -42,7 +42,7 @@ export const WidgetContentRenderer = ({
     case WidgetType.NOTES:
       return <NoteWidget widget={widget} />;
 
-    case WidgetType.RICH_TEXT:
+    case WidgetType.FIELD_RICH_TEXT:
       return <RichTextWidget widget={widget} />;
```

Comment:
Let’s rename `RichTextWidget` to `FieldRichTextWidget`. I realized I misread a part of your PR earlier.